### ISSUE TITLE
Table caption is not announced by some screen readers

### ISSUE THREAD

**Issue by lkszzajac** [ckeditor/ckeditor5#17179]
## 📝 Provide detailed reproduction steps (if any)

1. Go to documentation demo for table captions https://ckeditor.com/docs/ckeditor5/latest/features/tables/tables-caption.html
2. Focus the table and wait for screen reader announcement
3. Enter the table via keyboard (Enter key) and wait for screen reader announcement

### ✔️ Expected result

- table caption and dimensions are announced by the screen reader

### ❌ Actual result

- none of tested screen readers (VoiceOver, NVDA, JAWS) are announcing the table caption when focusing entire widget
- only NVDA is reading table dimensions and caption upon entry

## 📃 Other details

* Browser: Chrome
* OS: macOS/Windows
* First affected CKEditor version: n/a
* Installed CKEditor plugins: n/a
* Related issue: https://github.com/ckeditor/ckeditor5/issues/16438

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by dogawaf** [comment]
Related to https://github.com/ckeditor/ckeditor5/issues/15979

**Comment by CarmenHilpert** [comment]
Fixed with v46.1.0
